packages/composites/apy-finance/test-payload.js

```diff
@@ -0,0 +1,28 @@
+const environmentVariables = [
+  {envKey: 'AMBERDATA_DATA_PROVIDER_URL', defaultValue: 'amberdata'},
+  {envKey: 'COINAPI_DATA_PROVIDER_URL', defaultValue: 'coinapi'},
+  {envKey: 'COINGECKO_DATA_PROVIDER_URL', defaultValue: 'coingecko'},
+  {envKey: 'COINMARKETCAP_DATA_PROVIDER_URL', defaultValue: 'coinmarketcap'},
+  {envKey: 'COINPAPRIKA_DATA_PROVIDER_URL', defaultValue: 'coinpaprika'},
+  {envKey: 'CRYPTOCOMPARE_DATA_PROVIDER_URL', defaultValue: 'cryptocompare'},
+  {envKey: 'KAIKO_DATA_PROVIDER_URL', defaultValue: 'kaiko'},
+  {envKey: 'NOMICS_DATA_PROVIDER_URL', defaultValue: 'nomics'},
+]
+
+function searchEnvironment() {
+  for (const {envKey, defaultValue} of environmentVariables) {
+    const isSetEnvVar = process.env[envKey]
+    if (isSetEnvVar) return defaultValue
```

Comment:
We should warn log here that we're using a default var

Reply:
Oh it's not actually a default var per say. It's if there _is_ an environment variable set we want to return the value.

Like it needs to return `coingecko` if we find `COINGECKO_DATA_PROVIDER_URL`, no mater what `COINGECKO_DATA_PROVIDER_URL` may be.

Reply:
oh interesting, alright

Reply:
Maybe we should just call it `value`?

Reply:
Yeah